docker/service.go

```diff
@@ -273,3 +279,13 @@ func (s *Service) Containers() ([]project.Container, error) {
 
 	return result, nil
 }
+
+func (s *Service) SpecificiesHostPort() bool {
+	for _, port := range s.serviceConfig.Ports {
+		if strings.ContainsRune(port, ':') {
+			return true
+		}
+	}
```

Comment:
I think we need to fix this check in `docker-compose` because I don't think this is accurate. If ports is `127.0.0.1::3600` this will return true, but there is no host port specified.